### PR TITLE
Add dropdown for dimension order to Segmentation job card

### PR DESCRIPTION
**Pull request by elaubsch**
This PR adds a dropdown to only the Segmentation job card with the dimension order options: bxy, cxy, bxyc, and cxyb. This change addresses the issue where multi-batch image stacks with no channel dimension were previously handled as multi-channel stacks by the segmentation consumer. The dimension order input from the user is now saved to the Redis data for each segmentation job with the variable name `dimension_order`.

### CODE REVIEW

**Review by msschwartz21 — COMMENTED** [review]
Is it possible to add tooltip help text over the dimension menu to give users some additional help/context?

File: src/Predict/JobForm/SegmentationForm.js [hunk 1]
```diff
@@ -7,6 +7,33 @@ import Typography from '@mui/material/Typography';
 import ResolutionDropdown from './ResolutionDropdown';
 import ChannelForm from './ChannelForm';
 import jobData from '../jobData';
+import FormControl from '@mui/material/FormControl';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+
+function DimensionOrderDropdown(props) {
+  const { value, onChange } = props;
+  const orders = ['bxy', 'cxy', 'bxyc', 'cxyb'];
```
Comment (msschwartz21) [inline src/Predict/JobForm/SegmentationForm.js:16]: I'm wondering if these options would look better in all caps in the dropdown menu, but that is clearly an subjective decision so I'll defer to your preference.

**Comment by msschwartz21** [comment]
Just wanted to check if it's intentional to merge this into multiple-visualizers? Or if the target should be switched to master?

File: src/Predict/JobForm/SegmentationForm.js (hunk 1 above)
Comment (tddough98) [reply src/Predict/JobForm/SegmentationForm.js:16]: I think it should be all caps too. I've been using OME-TIFF's dimension order as a comparison point and they use all caps

https://docs.openmicroscopy.org/ome-model/6.0.0/ome-tiff/specification.html#dimensionorder

File: src/Predict/JobForm/SegmentationForm.js [hunk 2]
```diff
@@ -20,6 +49,7 @@ export default function SegmentationForm({ jobDropdown, setJobForm }) {
   const [selectedChannels, setSelectedChannels] = useState([
     ...Array(channels.length).keys(),
   ]);
+  const [dimensionOrder, setDimensionOrder] = useState('bxy');
```
Comment (tddough98) [inline src/Predict/JobForm/SegmentationForm.js:52]: Another place to change to all caps if we choose to

**Comment by tddough98** [comment]
> Is it possible to add tooltip help text over the dimension menu to give users some additional help/context?

I can add the tooltip and make a PR into this branch. How does this text sound?

Set the dimension order when predicting with multiple channels or multiple batches.

I feel like it might be missing something

**Review by tddough98 — APPROVED** [review]

**Pull request merged**